src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  transition-duration: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title &:hover {
+    color: #34568b;
+  }
+  &__image {
+    margin-bottom: 40px;
+    object-fit: contain;
+  }
+
+  &__title {
+    color: $main-accent;
+    margin-bottom: 4px;
+    transition-duration: 0.3s;
+  }
+
+  &__code {
+    font-weight: $font-normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
```

Comment:
Use descriptive names for variables. Point 11 from Checklist.

Reply:
please change it too - use descriptive names for variables